AssetMappr/application/showMap_cb.py

```diff
@@ -46,6 +44,39 @@ def update_figure(chosen_recycling):
         # Nonlocal tells this nested function to access map_df from the outer function - otherwise throws an undefined error
         nonlocal map_df
 
+        categoryList = ["Community Centers", "Entertainment", "Financial Assistance", "Food Access",
```

Comment:
Why are we manually defining the category list here? Can't we use the 'asset_categories' list that is inputted into the showMap_cb function?